clodius/tiles/fasta.py

```diff
@@ -0,0 +1,260 @@
+from pyfaidx import Fasta
+import re
+import numpy as np
+import functools as ft
+import pandas as pd
+import logging
+import base64
+
+logger = logging.getLogger(__name__)
+
+TILE_SIZE = 1024
+
+
+def get_quadtree_depth(chromsizes):
+    tile_size_bp = TILE_SIZE
+    min_tile_cover = np.ceil(sum(chromsizes) / tile_size_bp)
+    return int(np.ceil(np.log2(min_tile_cover)))
+
+
+
+def natsort_key(s, _NS_REGEX=re.compile(r"(\d+)", re.U)):
```

Comment:
This function is defined here: https://github.com/higlass/clodius/blob/develop/clodius/tiles/bigwig.py#L37

Would you mind factoring it out into `clodius.tiles.utils`?

Reply:
I'll refactor the locations of all of these duplicate functions in the next commit.